### ISSUE TITLE
Styled(undefined) in React Developer Tools when wrapping Link component

### ISSUE THREAD

**Issue by richardkall** [styled-components/styled-components#448]
## Version
2.0.0-1

## Reproduction
http://www.webpackbin.com/EygFtNoDf

## Steps to reproduce
Wrap a `<Link>` component (react-router 4.0.0-alpha.6) as `styled(Link)`.

## Expected Behavior
In React Developer Tools it should be named `Styled(Link)`.

## Actual Behavior
It is named `Styled(undefined)`.

**Comment by k15a** [comment]
Can't reproduce this: http://www.webpackbin.com/4yiCJSiDM

**Comment by k15a** [comment]
Going to close this for now because I can't reproduce this locally and in webpackbin.

Feel free to reopen this if you

**Comment by richardkall** [comment]
Thanks for looking into this. I managed to reproduce it in WebpackBin, I had not set the correct React Router version. It happens with react-router@4.0.0-alpha.6. I've updated my example:
http://www.webpackbin.com/EygFtNoDf

**Comment by mujuni88** [comment]
I removed stage-0 from configuration and it worked. http://www.webpackbin.com/V187kfCuf

**Comment by mxstbr** [comment]
Yep, that's it. Still waiting for @xcoderzach to translate a PR to v2 before we can merge v1 back into it.

**Comment by mxstbr** [comment]
This should be fixed in v2. Feel free to open a new issue if you're still encountering problems!
